<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1400</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>930</x>
      <y>40</y>
      <width>431</width>
      <height>421</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="player2Label">
           <property name="text">
            <string>Player Team 2 :</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="team2"/>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QGraphicsView" name="team2shirt"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="player2PerfLabel">
           <property name="text">
            <string>Player 1 Performance :</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPlainTextEdit" name="player2Performance">
           <property name="maximumSize">
            <size>
             <width>250</width>
             <height>27</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="player2PerfButton">
           <property name="text">
            <string>Set</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QLabel" name="player2ResLabel">
           <property name="text">
            <string>Player 2 Results :</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPlainTextEdit" name="player2Results">
           <property name="maximumSize">
            <size>
             <width>290</width>
             <height>27</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="player2ResButton">
           <property name="text">
            <string>Set</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>40</y>
      <width>421</width>
      <height>421</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_6">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="player1Label">
           <property name="text">
            <string>Player Team 1:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="team1"/>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QGraphicsView" name="team1shirt"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <item>
          <widget class="QLabel" name="player1PerfLabel">
           <property name="text">
            <string>Player 1 Performance :</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPlainTextEdit" name="player1Performance">
           <property name="maximumSize">
            <size>
             <width>170</width>
             <height>27</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="player1PerfButton">
           <property name="text">
            <string>Set</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QLabel" name="player1ResLabel">
           <property name="text">
            <string>Player 1 Results :</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPlainTextEdit" name="player1Results">
           <property name="maximumSize">
            <size>
             <width>200</width>
             <height>27</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="player1ResButton">
           <property name="text">
            <string>Set</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>160</y>
      <width>321</width>
      <height>171</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_8">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_7">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_7">
         <item>
          <widget class="QLabel" name="gameDateLabel">
           <property name="text">
            <string>Date:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDateEdit" name="gameDate"/>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="gameStartButton">
       <property name="text">
        <string>Start</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="addTeam1PlayerButton">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>480</y>
      <width>421</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Add</string>
    </property>
   </widget>
   <widget class="QPushButton" name="addTeam2PlayerButton">
    <property name="geometry">
     <rect>
      <x>930</x>
      <y>480</y>
      <width>431</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Add</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>520</y>
      <width>181</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Number of players - team1: </string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>1090</x>
      <y>520</y>
      <width>181</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Number of players - team2: </string>
    </property>
   </widget>
   <widget class="QLabel" name="numberOfPlayersTeam1Label">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>520</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="numberOfPlayersTeam2Label">
    <property name="geometry">
     <rect>
      <x>1280</x>
      <y>520</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1400</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
